### PR TITLE
Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-generic.git`](https://github.com/cisagov/skeleton-generic.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

---

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
👍 👍 🐳

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -18,11 +18,11 @@ on:
       remote-shell:
         description: "Debug with remote shell"
         required: true
-        default: false
+        default: "false"
```
Comment (jsf9k) [inline .github/workflows/build.yml:21]: Why is this change necessary?

**Review by jsf9k — APPROVED** [review]
I have one minor question, but otherwise this all looks great.

File: .github/workflows/build.yml (hunk 1 above)
Comment (mcdonnnj) [reply .github/workflows/build.yml:21]: These values are expected to be string values because of how they are processed. Without the quotes it will be interpreted as a YAML boolean which probably doesn't _really_ matter, but it fails schema validation which is doing a strict interpretation so it needed to be coerced to a string value. I assume it's to avoid unexpected auto-conversion issues like trying to use `3.10` for the Python version to install and getting `3.1` instead.
